bin/oneoff/accessibility/accessibility_checker.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+require_relative '../../../deployment'
+require 'cdo/only_one'
+require 'json'
+
+PALLY_PATH = "../../../apps/node_modules/pa11y-ci/bin/pa11y-ci.js"
```

Comment:
this makes it so that you have to run the script from `bin/oneoff/accessibility` or pa11y can't be loaded. if you expand the path, you'll be able to run it from anywhere:

https://github.com/code-dot-org/code-dot-org/blob/c5f6bfe98f8e59c8b1715ce90c898dc8f6e50edd/bin/archive/messages_sent_to#L2

Reply:
This is convenient! Learned something new today!